index.json

```diff
@@ -465,11 +465,10 @@
     {
         "manifest": {
             "id": "org.stremio.local",
-            "version": "1.9.1",
+            "version": "1.10.0",
             "description": "Local add-on to find playable files: .torrent, .mp4, .mkv and .avi",
-            "name": "Local Files",
+            "name": "Local Files (without catalog support)",
```

Comment:
Are we sure that this change makes sense? Would it not change the name of the Local Files addon for those that have it installed with catalog support also?

Reply:
It is reloaded on starting the app.

Reply:
alright, if you're sure, we will need to ensure we test it properly though